### PR TITLE
Feat/regstration page

### PR DESCRIPTION
**Pull request by seanlawrenz**
Sets up the registration page

### CODE REVIEW

File: client/src/Modules/Family/FamilyContainer.js [hunk 1]
```diff
@@ -1,207 +1,46 @@
...
-                                            </div>
-                                        </div>
-                                    </div>
-                                </form>
-
-
-                            </div>
-                        </div>
-                    </div>
-                </section>
```
Comment (seanlawrenz) [inline client/src/Modules/Family/FamilyContainer.js:200]: Much of the form logic is unnecessary with the addition of `react-hook-form`

File: client/src/Modules/Family/__test__/familyContainer.test.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  expect(() => {
+    render(
+      <Router>
+        <FamilyContainer />
+      </Router>
+    );
+  }).not.toThrowError();
+});
+
+test(`should show 'Passwords should match!' if passwords do not match`, async () => {
```
Comment (seanlawrenz) [inline client/src/Modules/Family/__test__/familyContainer.test.js:16]: This is not implemented at `FamilyContainer` but rather is down stream at `PrimaryInfoFormComponent`. However testing this here follows more of what the user would do versus testing within the component.

**Comment by davidpickering** [comment]
Hey @seanlawrenz, great job translating those wireframes into UI! There are a number of updates largely stemming from difficulty communicating to the designers (they are good, but there is only so much you can type out or hash out through zoom). The wireframes are close, but not 100% gospel. So, a few comments to you should move us that last 10%:

Comments in the PowerPoint, based off images shared by Sean

Here is a summary of changes:
1. A reorder of the components: (Top to Bottom). There are more details in the attached PowerPoint.
2. Minor text modifications in some components
3. Addition of family members components as an optional step 1.5


[Feedback.pptx](https://github.com/midohiofoodbank/freshtrak-public/files/4513484/Feedback.pptx)

**Comment by nvengal** [comment]
Looks like there are some conflicts from the large frontend clean up pr. That wasn't meant to change functionality on these components, so when you resolve the conflicts you should be fine with just keeping your changes.

**Comment by nvengal** [comment]
@seanlawrenz 
In preparation for adding another team of contributors to the project we're trying to move away from the monorepo structure and give each project it's own repository. Do you mind moving this PR over to https://github.com/midohiofoodbank/freshtrak-client?

**Comment by seanlawrenz** [comment]
Closing as this PR is being moved to the new repo

**Pull request closed without merging**